customcontext/texture/hybristexture.cpp

```diff
@@ -272,11 +272,10 @@ NativeBuffer *NativeBuffer::create(const QImage &image)
     return buffer;
 }
 
-HybrisTexture::HybrisTexture(NativeBuffer *buffer)
+HybrisTexture::HybrisTexture(QSharedPointer<NativeBuffer> buffer)
```

Comment:
Indeed it might, but as I don't have any means of testing it I'd rather not try to fix it :)